Demos/Blazorise.Demo/Pages/Tests/DataGridPage.razor

```diff
@@ -319,7 +319,7 @@
                                 }
                             </DisplayTemplate>
                             <EditTemplate>
-                                <Select TValue="string" SelectedValue="@((string)( context.CellValue ))" SelectedValueChanged="@(( v ) => context.CellValue = v)">
+                                <Select TValue="string" SelectedValue="@((string)( context.CellValue ))" SelectedValueChanged="@(( v ) => { context.CellValue = v; context.UpdateCell( nameof( Employee.FirstName ), "Mr" +(v=="F"?"s":"") + context.Item.FirstName ); })">

```

Comment:
Haven't tested. But isn't it missing a space between Mr/Mrs and the name?

Reply:
It's just for internal purposes. I will revert it before merge.

Reply:
Could leave it... It's always good to have examples of the various features.

Reply:
In that case I need to make it better :)